backend/src/data/models/user/user.model.ts

```diff
@@ -28,6 +28,22 @@ class User extends Abstract {
           to: `${DbTableName.GROUPS}.id`,
         },
       },
+      interviewee: {
+        relation: Model.HasManyRelation,
+        modelClass: Interview,
+        join: {
+          from: `${DbTableName.USERS}.id`,
+          to: `${DbTableName.INTERVIEWS}.interviewee_user_id`,
+        },
+      },
+      interviewer: {
+        relation: Model.HasManyRelation,
+        modelClass: Interview,
+        join: {
+          from: `${DbTableName.USERS}.id`,
+          to: `${DbTableName.INTERVIEWS}.interviewer_user_id`,
+        },
+      },
```

Comment:
Don't think that we would use this relation in user's model

Reply:
done